lib/rpmts.c

```diff
@@ -378,13 +378,18 @@ static void loadKeyring(rpmts ts)
     /* Never load the keyring if signature checking is disabled */
     if ((rpmtsVSFlags(ts) & RPMVSF_MASK_NOSIGNATURES) !=
 	RPMVSF_MASK_NOSIGNATURES) {
+	char *kt = rpmExpand("%{?_keyring}", NULL);
```

Comment:
I'm not a fan of cryptic variables like `kt`, but we do this so much already... I debated whether I should ask to not make this worse, but ehh... In any case, here's a _soft_ ask to consider making this a clearer variable, but not a hard requirement.

Reply:
I'm a firm believer in many of kernel coding style principles, including naming: https://www.kernel.org/doc/html/v5.11/process/coding-style.html#naming 

It's a local temporary variable and doesn't carry any great big logic in it ... but making the name a *little* longer doesn't hurt and besides, you asked nicely :innocent: